tools/clang/test/HLSL/out-param-diagnostics.hlsl

```diff
@@ -1,22 +1,22 @@
 // RUN: %clang_cc1 -fsyntax-only -verify %s
 
-void UnusedEmpty(out int Val) {} // expected-warning{{parameter 'Val' is uninitialized when used here}} expected-note{{variable 'Val' is declared here}}
+void UnusedEmpty(out int Val) {} // expected-note {{variable 'Val' is declared here}} expected-warning {{parameter 'Val' is uninitialized when used here}} fxc-pass {{}}
 
 // Neither of these should warn
 void UnusedInAndOut(in out int Val) {}
 void UnusedInOut(inout int Val) {}
 
 
-int Returned(out int Val) { // expected-note{{variable 'Val' is declared here}}
-  return Val; // expected-warning{{parameter 'Val' is uninitialized when used here}}
+int Returned(out int Val) { // expected-note {{variable 'Val' is declared here}} fxc-pass {{}}
```

Comment:
I don't think the FXC diagnostics in this file provide any benefit, so we should probably exclude this.